src/components/FeaturedProducts/styles.js

```diff
@@ -1,41 +1,76 @@
-import { makeStyles } from '@material-ui/core';
+import { makeStyles } from "@material-ui/core";
 
-const backGround = '#f6f9fc';
 
 const useStyles = makeStyles((theme) => ({
   root: {
-    display: 'flex',
-    flexDirection: 'column',
-    backgroundColor: backGround,
-    height: '450px',
-    justifyContent: 'center',
-    [theme.breakpoints.down('lg')]: {
-      display: 'flex',
-    }
+    display: "flex",
+    flexDirection: "column",
+    backgroundColor: theme.palette.main.blue,
+    height: "100%",
+    justifyContent: "center",
+    marginBottom: "2rem",
+    [theme.breakpoints.down("lg")]: {
+      display: "100%",
```

Comment:
maybe you mean width: 100%, don't you?

Reply:
sorry! before i said "display:" none "" and forgot to delete that. It was corrected!